build/AzurePipelinesTemplates/WindowsAppSDK-RunHelixTests-Job.yml

```diff
@@ -93,21 +96,20 @@ jobs:
       nugetConfigPath: nuget.config
       restoreDirectory: packages
 
-  # Download the MRT Core test binaries
+  # Download the BuildOutput from the Build stage(s).
+  # We only bring down the relevant content for this build config (Debug/Release) & platform, to save some space and time.
   - task: DownloadBuildArtifacts@0
-    displayName: 'Download: Binaries'
+    displayName: 'Download: BuildOutput'
     inputs:
-      artifactName: 'mrtcore_test_binaries_$(buildPlatform)'
+      artifactName: 'BuildOutput'
       downloadPath: $(testPayloadDir)
+      itemPattern: 'BuildOutput/$(buildConfiguration)/$(buildPlatform)/**/*'
 
-  # Publish the full testPayloadDir contents.
-  # This is not required, but can be useful for debugging purposes.
-  - task: PublishBuildArtifacts@1
-    displayName: 'Publish testPayloadDir'
-    condition: always()
-    inputs:
-      PathtoPublish: $(testPayloadDir)
-      artifactName: HelixTestPayloadDir_$(buildPlatform)_$(buildConfiguration)
+  - powershell: |
+      Write-Host ""
+      Write-Host "$(Build.SourcesDirectory)"
+      Tree /F /A $(Build.SourcesDirectory)
+    displayName: 'DIAG: dir'
```

Comment:
?

Reply:
Ah, sorry about this. I had added this to double-check that the full BuildOutput was being downloaded. I'll remove it.